### PR TITLE
Fix Edit Skill in SkillCreator and bugs #2519

### LINKED ISSUE #2519: Fix Edit Skill in SkillCreator and bugs
**Actual Behaviour**

On editing a skill, the skill code is not correct in Editor

**Expected Behaviour**

Fix it

**Steps to reproduce it**

https://susi.ai/Novelty%20and%20Humour/joke/edit/en

**Screenshots of the issue**

Screenshot 2019-07-12 at 12 37 37 AM

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2519, #2534

Changes: 
- [x] Fix Edit Skill in SkillCreator
- [x] UI Design view for boatbuilder edit
- [x] Fix undo typo in skill, delete icon color
- [x] Dashboard: Different size of gray area padding
- [x] Fix CircleImage alignment in avatars
- [x] Fix store draft
- [x] Fix loader
- [x] Fix undo typo in skill, delete icon color
- [x] Fix design UI avatar and background image on edit, saved it on the backend

Demo Link: https://pr-2533-fossasia-susi-web-chat.surge.sh

Screenshots of the change: 

Screenshot 2019-07-13 at 4 15 17 AM

Screenshot 2019-07-13 at 7 44 08 PM

### CODE REVIEW

**Comment by harsh-jindal** [comment]
@fragm3 On edit skill Image does not show up.
Screenshot 2019-07-13 at 8 42 04 AM

Also a couple of more bugs:
1. Save draft does not work.
2. If you change the bot avatar for preview and save, the changes are not reflected when you open again.

File: src/components/cms/BotBuilder/BotBuilderPages/DesignViews/UIView.js [hunk 1]
```diff
@@ -281,14 +278,40 @@ class UIView extends Component {
...
+    form.append('access_token', accessToken);
+    form.append('image_name', filename);
+    form.append('image', file);
+    uploadImage(form)
+      .then(payload => {
+        this.setState({
+          uploadingBotbuilderIconImg: false,
+        });
+        actions.setBotAvatar({
+          botbuilderIconImg: `https://api.susi.ai/cms/getImage.png?image=${payload.imagePath}`,
```
Comment (akshatnitd) [inline src/components/cms/BotBuilder/BotBuilderPages/DesignViews/UIView.js]: Do not hardcode the server url

Comment (akshatnitd) [reply src/components/cms/BotBuilder/BotBuilderPages/DesignViews/UIView.js]: Can you use the getImage util here?

File: src/components/cms/BotBuilder/BotBuilderPages/DesignViews/UIView.js [hunk 2]
```diff
@@ -217,17 +221,22 @@ class UIView extends Component {
...
-        botbuilderBodyBackgroundImgName: imageName,
-      })
+    let form = new FormData();
+    form.append('access_token', accessToken);
+    form.append('image_name', this.backgroundImageSelectedFile.name);
+    form.append('image', this.backgroundImageSelectedFile);
+    uploadImage(form)
       .then(payload => {
+        this.setImageBodyBackground(
+          `https://api.susi.ai/cms/getImage.png?image=${payload.imagePath}`,
```
Comment (akshatnitd) [inline src/components/cms/BotBuilder/BotBuilderPages/DesignViews/UIView.js]: Do not hardcode the server url

File: src/components/cms/BotBuilder/BotBuilderPages/DesignViews/UIView.js (hunk 1 above)
Comment (fragm3) [reply src/components/cms/BotBuilder/BotBuilderPages/DesignViews/UIView.js]: @akshatnitd The irony here is yesterday only I made a PR for getImageUrl

**Comment by fragm3** [comment]
@akshatnitd made the changes.

**Pull request merged**